docs/modules/operation/pages/deep-dive/admin/housekeeping/introduction.adoc

```diff
@@ -1,46 +1,29 @@
 
-[[housekeeping]]
 = Housekeeping Tasks
 
-There are a number of housekeeping tasks you may want to do regularly to ensure optimum system performance.
+There are a number of housekeeping tasks that you may want to do regularly to ensure optimum system performance.
 We also recommend you complete some of these housekeeping tasks before upgrading {page-component-title}.
 
 == Prior to upgrading
 
 [[prune-events]]
 === Prune unneeded events
+
 Use vacuumd or cron to regularly prune unneeded events.
 This helps to clean up your events table, and reduces the amount of data in your backups.
 We also recommend doing this before you begin the {page-component-title} upgrade process.
 
-For example, run the following to delete events older than six weeks that have no associated outages:
-
-[source, sql]
-----
---# this deletes any events that are not associated with outages
-DELETE FROM events WHERE NOT EXISTS
-(SELECT svclosteventid FROM outages WHERE svclosteventid = events.eventid
-UNION
-SELECT svcregainedeventid FROM outages WHERE svcregainedeventid = events.eventid
-UNION
-SELECT eventid FROM notifications WHERE eventid = events.eventid)
-AND eventtime < now() - interval '6 weeks';
-----
-
-== After Upgrading
+== After upgrading
```

Comment:
This is an empty header with no content.

Reply:
It's the same as the "Prior to upgrading" heading. It is unusual that there is only one item under each, but it's a way to separate the pre- and post- upgrading tasks visually.